example/ooni/oorchestrate.cpp

```diff
@@ -1,94 +0,0 @@
-// Part of Measurement Kit <https://measurement-kit.github.io/>.
-// Measurement Kit is free software under the BSD license. See AUTHORS
-// and LICENSE for more information on the copying conditions.
-
-#include "src/libmeasurement_kit/common/fcompose.hpp"
-#include <measurement_kit/ooni.hpp>
-
-#include <unistd.h>
-
-#include <future>
-
-using namespace mk::ooni::orchestrate;
-using namespace mk::ooni;
-using namespace mk;
-
-#define USAGE "oorchestrate [-v]"
-
-int main(int argc, char **argv) {
```

Comment:
Removed the example client because (i) it was complicating the Windows build and (ii) we're redoing this API.